scripts/android_package.sh

```diff
@@ -117,10 +117,10 @@ print Y "Importing translation files..."
 python3 scripts/importLanguages.py $([[ "$PROD" ]] && echo "-p" || echo "") || die "Failed to import languages"
 
 printn Y "Computing the version... "
-SHORTVERSION=$(cat version.pri | grep VERSION | grep defined | cut -d= -f2 | tr -d \ )
+export SHORTVERSION=$(cat version.pri | grep VERSION | grep defined | cut -d= -f2 | tr -d \ ) # Export so gradle can pick it up
+export VERSIONCODE=$(echo $SHORTVERSION| tr -d .)"0"
```

Comment:
This is OK if you add a comment in version.pri saying something like:
"version numbers must be <= 9!" or a single-digit number... 

otherwise, version=2.0.12 will generate a "20120" and that is so wrong.